draft-ietf-lake-edhoc.md

```diff
@@ -787,9 +787,7 @@ where
 PRK_exporter  = EDHOC-KDF( PRK_out, 10, h'', hash_length )
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-where hash_length denotes the output size in bytes of the EDHOC hash algorithm of the selected cipher suite.
-
-PRK_exporter MUST be derived anew from PRK_out if EDHOC-KeyUpdate is used, see {{keyupdate}}.
+where hash_length denotes the output size in bytes of the EDHOC hash algorithm of the selected cipher suite. Note that PRK_exporter changes everytime EDHOC-KeyUpdate is used, see {{keyupdate}}.
```

Comment:
"Note that PRK_exporter changes everytime EDHOC-KeyUpdate is used, see {{keyupdate}}."

{{keyupdate}} does not describe this change.
everytime -> every time